### DISCUSSION TITLE
[WEEKLY DROP] War Bounds

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1593] [General]
This is already the second week that you guys don't deliver the weekly rewards. Without official announcement from dev team. This affects players who depend on this WB to make their changes. We know how busy you are with the land expansion but it seems that we have been forgotten and it causes insecurity about communication.
Could you please look at this?
Thank you.

**Comment by future2back** [comment]
Just received first week, when will be the next one? It also afects our sync strategy. The lack of communication makes me very intrigued, I don't know what strategy to follow. What will be done with the remaining WB? Are you going to encourage people who have a lot to spend what they still have in war tent? What happens to those under 300? Same question can be applied to jack-o-lantern... Will you find a way to recycle this? Will who have it lose it? Would it be a good strategy to have extras beyond is needed?

Could you please clarify and improve communication? We are getting announcements of new things and forgetting current things, this can be confusing.

Thank you.

**Reply by future2back** [reply]
@adamhannigan the 150 WB and fertilizers were from two weeks ago? Last week's reward is missing, right? Could you please update this? It's affecting the sync... Thank you!

**Reply by adamhannigan** [reply]
@future2back Apologies, this post wasn't surfaced until now.

There have been some announcements surrounding the war over the past 2 weeks that hopefully satisfy your questions.

As per the announcement the remaining weeks of the war will not have 'team rewards' as everyone is fighting together 💪
